### ISSUE TITLE
ECL: Marking rtest16/test step 50 as "known bad" doesn't work 

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3185]
Imported from SourceForge on 2024-07-07 15:55:02
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2018-12-29 08:45:10
Original: https://sourceforge.net/p/maxima/bugs/3514

---

rtest16/step 50 sporadically fails in ECL. Even if it is marked as "known bad" for some reason it counts as an unexpected error:
~~~
makelist(block([fpprintprec : m], makelist(string(foo 10 ), n, - 10, 10)), m, 2, 20))], map(lambda([s1, s2], if sequalignore(s1, s2) then true else s2 # s1), flatten(L1), flatten(L2)), delete(true, %%)) Result: [12.3e-11 # 1.2E-10, 12.3e-10 # 1.2E-9, 12.3e-9 # 1.2E-8, 12.3e-8 # 1.2E-7, 12.3e-7 # 1.2E-6, 12.3e-6 # 1.2E-5, 12.3e-5 # 1.2E-4, 12.34e-11 # 1.23E-10, 12.34e-10 # 1.23E-9, 12.34e-9 # 1.23E-8, 12.34e-8 # 1.23E-7, 12.34e-7 # 1.23E-6, 12.34e-6 # 1.23E-5, 12.34e-5 # 1.23E-4, 12.344e-11 # 1.234E-10, 12.344e-10 # 1.234E-9, 12.344e-9 # 1.234E-8, 12.344e-8 # 1.234E-7, 12.344e-7 # 1.234E-6, 12.344e-6 # 1.234E-5, 12.344e-5 # 1.234E-4, 12.3443e-11 # 1.2344E-10, 12.3443e-10 # 1.2344E-9, 12.3443e-9 # 1.2344E-8, 12.3443e-8 # 1.2344E-7, 12.3443e-7 # 1.2344E-6, 12.3443e-6 # 1.2344E-5, 12.3443e-5 # 1.2344E-4, 12.34432e-11 # 1.23443E-10, 12.34432e-10 # 1.23443E-9, 12.34432e-9 # 1.23443E-8, 12.34432e-8 # 1.23443E-7, 12.34432e-7 # 1.23443E-6, 12.34432e-6 # 1.23443E-5, 12.34432e-5 # 1.23443E-4, 12.344321e-11 # 1.234432E-10, 12.344321e-10 # 1.234432E-9, 12.344321e-9 # 1.234432E-8, 12.344321e-8 # 1.234432E-7, 12.344321e-7 # 1.234432E-6, 12.344321e-6 # 1.234432E-5, 12.344321e-5 # 1.234432E-4, 12.3443211e-11 # 1.2344321E-10, 12.3443211e-10 # 1.2344321E-9, 12.3443211e-9 # 1.2344321E-8, 12.3443211e-8 # 1.2344321E-7, 12.3443211e-7 # 1.2344321E-6, 12.3443211e-6 # 1.2344321E-5, 12.3443211e-5 # 1.2344321E-4, 12.34432112e-11 # 1.23443211E-10, 12.34432112e-10 # 1.23443211E-9, 12.34432112e-9 # 1.23443211E-8, 12.34432112e-8 # 1.23443211E-7, 12.34432112e-7 # 1.23443211E-6, 12.34432112e-6 # 1.23443211E-5, 12.34432112e-5 # 1.23443211E-4, 12.344321123e-11 # 1.234432112E-10, 12.344321123e-10 # 1.234432112E-9, 12.344321123e-9 # 1.234432112E-8, 12.344321123e-8 # 1.234432112E-7, 12.344321123e-7 # 1.234432112E-6, 12.344321123e-6 # 1.234432112E-5, 12.344321123e-5 # 1.234432112E-4, 12.3443211234e-11 # 1.2344321123E-10, 12.3443211234e-10 # 1.2344321123E-9, 12.3443211234e-9 # 1.2344321123E-8, 12.3443211234e-8 # 1.2344321123E-7, 12.3443211234e-7 # 1.2344321123E-6, 12.3443211234e-6 # 1.2344321123E-5, 12.3443211234e-5 # 1.2344321123E-4, 12.34432112344e-11 # 1.23443211234E-10, 12.34432112344e-10 # 1.23443211234E-9, 12.34432112344e-9 # 1.23443211234E-8, 12.34432112344e-8 # 1.23443211234E-7, 12.34432112344e-7 # 1.23443211234E-6, 12.34432112344e-6 # 1.23443211234E-5, 12.34432112344e-5 # 1.23443211234E-4, 12.344321123443e-11 # 1.234432112344E-10, 12.344321123443e-10 # 1.234432112344E-9, 12.344321123443e-9 # 1.234432112344E-8, 12.344321123443e-8 # 1.234432112344E-7, 12.344321123443e-7 # 1.234432112344E-6, 12.344321123443e-6 # 1.234432112344E-5, 12.344321123443e-5 # 1.234432112344E-4, 12.3443211234432e-11 # 1.2344321123443E-10, 12.3443211234432e-10 # 1.2344321123443E-9, 12.3443211234432e-9 # 1.2344321123443E-8, 12.3443211234432e-8 # 1.2344321123443E-7, 12.3443211234432e-7 # 1.2344321123443E-6, 12.3443211234432e-6 # 1.2344321123443E-5, 12.3443211234432e-5 # 1.2344321123443E-4, 12.34432112344321e-11 # 1.23443211234432E-10, 12.34432112344321e-10 # 1.23443211234432E-9, 12.34432112344321e-9 # 1.23443211234432E-8, 12.34432112344321e-8 # 1.23443211234432E-7, 12.34432112344321e-7 # 1.23443211234432E-6, 12.34432112344321e-6 # 1.23443211234432E-5, 12.34432112344321e-5 # 1.23443211234432E-4, 12.344321123443212e-11 # 1.234432112344321E-10, 12.344321123443212e-10 # 1.234432112344321E-9, 12.344321123443211e-9 # 1.234432112344321E-8, 12.344321123443212e-8 # 1.234432112344321E-7, 12.34432112344321e-7 # 1.234432112344321E-6, 12.344321123443213e-6 # 1.234432112344321E-5, 12.344321123443212e-5 # 1.234432112344321E-4, 12.344321123443212e-11 # 1.234432112344321E-10, 12.344321123443212e-10 # 1.234432112344321E-9, 12.344321123443211e-9 # 1.234432112344321E-8, 12.344321123443212e-8 # 1.234432112344321E-7, 12.34432112344321e-7 # 1.234432112344321E-6, 12.344321123443213e-6 # 1.234432112344321E-5, 12.344321123443212e-5 # 1.234432112344321E-4, 12.344321123443212e-11 # 1.234432112344321E-10, 12.344321123443212e-10 # 1.234432112344321E-9, 12.344321123443211e-9 # 1.234432112344321E-8, 12.344321123443212e-8 # 1.234432112344321E-7, 12.34432112344321e-7 # 1.234432112344321E-6, 12.344321123443213e-6 # 1.234432112344321E-5, 12.344321123443212e-5 # 1.234432112344321E-4, 12.344321123443212e-11 # 1.234432112344321E-10, 12.344321123443212e-10 # 1.234432112344321E-9, 12.344321123443211e-9 # 1.234432112344321E-8, 12.344321123443212e-8 # 1.234432112344321E-7, 12.34432112344321e-7 # 1.234432112344321E-6, 12.344321123443213e-6 # 1.234432112344321E-5, 12.344321123443212e-5 # 1.234432112344321E-4, 12.344321123443212e-11 # 1.234432112344321E-10, 12.344321123443212e-10 # 1.234432112344321E-9, 12.344321123443211e-9 # 1.234432112344321E-8, 12.344321123443212e-8 # 1.234432112344321E-7, 12.34432112344321e-7 # 1.234432112344321E-6, 12.344321123443213e-6 # 1.234432112344321E-5, 12.344321123443212e-5 # 1.234432112344321E-4] This differed from the expected result: [] 681/682 tests passed (not counting 2 expected errors) The following 1 problem failed: (50) 
~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 15:55:03
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2018-12-31 19:15:52
Original: https://sourceforge.net/p/maxima/bugs/3514/#8825

---

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 15:55:06
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2018-12-31 19:15:52
Original: https://sourceforge.net/p/maxima/bugs/3514/#d91e

---

Resolved with commit  94d663c286
